traceInfo.go

```diff
@@ -0,0 +1,52 @@
+package zipkin
+
+import "github.com/elodina/go-avro"
+
+type TraceInfo struct {
+	SpanId       int64
+	TraceId      int64
+	ParentSpanId interface{}
+	Sampled      bool
+}
+
+func NewTraceInfo() *TraceInfo {
+	return &TraceInfo{
+		Sampled: false,
+	}
+}
+
+func (this *TraceInfo) Schema() avro.Schema {
+	if _TraceInfo_schema_err != nil {
+		panic(_TraceInfo_schema_err)
+	}
+	return _TraceInfo_schema
+}
+
+// Generated by codegen. Please do not modify.
+var _TraceInfo_schema, _TraceInfo_schema_err = avro.ParseSchema(`{
+    "type": "record",
+    "name": "TraceInfo",
+    "fields": [
+        {
+            "name": "spanId",
+            "type": "long"
+        },
+        {
+            "name": "traceId",
+            "type": "long"
+        },
+        {
+            "name": "parentSpanId",
+            "default": null,
+            "type": [
+                "null",
+                "long"
+            ]
+        },
+        {
+            "name": "sampled",
```

Comment:
note there's also a debug flag that's possible. In brave, we recently moved to the binary form that finagle has been using: https://github.com/openzipkin/brave/blob/master/brave-core/src/main/java/com/github/kristofa/brave/SpanId.java

Do you know any other things that are propagating with avro serialization? (curious)